terraform/modules/gost_api/storage.tf

```diff
@@ -23,3 +23,37 @@ module "efs_data_volume" {
     }
   }
 }
+
+module "arpa_audit_reports_bucket" {
```

Comment:
@TylerHendrickson can you confirm if this is the right place to add the s3 bucket definition in order for it to get created in AWS?

I'm still working on whether the specific configurations is what we need. I'm basing this off of `grants_source_data` bucket which I think is close to what we want to achieve with ARPA reports.

Once I get this setup in my `awslocal` I'll open up this draft PR to a live PR.

Reply:
Yep, perfect.